zio-http/src/main/scala/zhttp/http/middleware/Web.scala

```diff
@@ -171,6 +173,31 @@ private[zhttp] trait Web extends Cors with Csrf with Auth with HeaderModifier[Ht
       _ => middleware,
       _ => Middleware.identity,
     )
+
+  /**
+   * Removes the trailing slash from the path.
+   */
+  final def trailingSlashDrop[R, E]: HttpMiddleware[R, E] =
+    Middleware.identity[Request, Response].contramap[Request](removeTrailingSlash)
```

Comment:
```suggestion
    Middleware.identity[Request, Response].contramap[Request](_.dropTrailingSlash)
```

Will need to add `dropTrailingSlash` in `Request`